src/Pages/ContactPage.jsx

```diff
@@ -1,78 +1,95 @@
 import React from "react";
+import Toastify from 'toastify-js';
+import "toastify-js/src/toastify.css";
 import BackBtn from "../components/BackBtn/BackBtn";
 
 function ContactPage() {
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    // Trigger a Toastify notification
+    Toastify({
+      text: "Message sent successfully!",
```

Comment:
Replace the text to "Thanks for contacting us. We care for you and will get back to you soon !"